src/components/manage/Blocks/Video/schema.js

```diff
@@ -0,0 +1,50 @@
+import { defineMessages } from 'react-intl';
+
+const messages = defineMessages({
+  Video: {
+    id: 'Video',
+    defaultMessage: 'Video',
+  },
+  VideoURL: {
+    id: 'Video URL',
+    defaultMessage: 'Video URL',
+  },
+  Preview_image: {
+    id: 'Preview Image URL',
+    defaultMessage: 'Preview Image URL',
+  },
+  Alignment: {
+    id: 'Alignment',
+    defaultMessage: 'Alignment',
+  },
+});
+export const VideoBlockSchema = (props) => ({
+  title: props.intl.formatMessage(messages.Video),
+  block: 'Video',
+  fieldsets: [
+    {
+      id: 'default',
+      title: 'Default',
+      fields: ['url', 'preview_image', 'align'],
+    },
+  ],
+
+  properties: {
+    url: {
+      title: props.intl.formatMessage(messages.VideoURL),
```

Comment:
Just realised that the original has an action item with the X to remove the contents of the field:

<img width="389" alt="image" src="https://user-images.githubusercontent.com/486927/169698935-b18bdee7-c6b0-4c2f-83f2-416d151b66d7.png">

In fact, it happens also in the new image block refactor /cc @nileshgulia1 @ThomasKindermann 

I'd like to maintain it somehow... maybe add it to the URLWidget?